src/main/java/org/cascadebot/cascadebot/commands/developer/EvalCommand.java

```diff
@@ -92,7 +92,8 @@ public void onCommand(Member sender, CommandContext context) {
                 String codeToRun = imports + " " + code;
                 String results = String.valueOf(scriptEngine.eval(codeToRun));
                 if (results.isBlank()) results = "Empty result!";
-                PasteUtils.pasteIfLong(results, 2048, context::reply);
+                // The value of 1950 is the max limit of 2000 characters with a 50 character buffer
+                PasteUtils.pasteIfLong(results, 1950, context::reply);
```

Comment:
We should start storing these common numbers in a specific class, cause we use them everywhere and if discord changes limits it will become much easier to change. Doesn't have to change on this pr, but something to note we should do.